wallet/src/main/java/com/android/identity_credential/wallet/ui/prompt/biometric/BiometricPrompt.kt

```diff
@@ -8,6 +8,53 @@ import androidx.biometric.BiometricPrompt.PromptInfo
 import androidx.core.content.ContextCompat
 import androidx.fragment.app.FragmentActivity
 import com.android.identity.android.securearea.UserAuthenticationType
+import com.android.identity_credential.wallet.R
+import kotlinx.coroutines.suspendCancellableCoroutine
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+
+/**
+ * [showBiometricPrompt]
+ * Async extension function of a [FragmentActivity] to show Biometric Prompt and return the result as a [Boolean] of whether
+ * authentication was successful, or raises/throws an Exception, such as [IllegalStateException],
+ * if an error prevented the Biometric Prompt from showing.
+ *
```

Comment:
The first line looks weird. In general this should be line JavaDoc where the first line in a single-line short descriptive sentence, then you have a blank line, then one or more paragraphs of prose. So in this case

```
/**
 * Show a biometric prompt
 *
 * This suspend function shows a a biometric prompt and [...]
```

Ditto in other places.